src/theory/quantifiers/cegqi/ceg_arith_instantiator.cpp

```diff
@@ -262,7 +262,7 @@ bool ArithInstantiator::processAssertion(CegInstantiator* ci,
       }
       else
       {
-        Assert(d_type.isReal());
+        Assert(d_type.isArithmetic());
```

Comment:
```suggestion
        Assert(d_type.isReal());
```

The integer case is handled above.